share/spice/color_picker/color_picker.css

```diff
@@ -0,0 +1,91 @@
+#color_picker_container {
```

Comment:
Convention is to prepend CSS selectors with `.zci--color_picker`. Not essential since you have `#color_picker_container` but just so you're aware. :)